kvirt/k3s/workers.yml

```diff
@@ -20,15 +20,8 @@
  pool: {{ pool }}
  numcpus: {{ worker_numcpus | default(numcpus, numcpus) }}
  memory: {{ worker_memory | default(memory, memory) }}
- files:
-  - path: /root/join.sh
-    origin: {{ basedir }}/clusters/{{ cluster }}/join.sh
- cmds:
- - apt-get -y install curl
-{% if sdn == 'cilium' %}
- - mount bpffs -t bpf /sys/fs/bpf
-{% endif %}
- - bash /root/join.sh
+ scripts:
+  - join.sh
```

Comment:
the path should be kept the same, otherwise this wont work when scaling (and it wouldnt on a new install even, i suspect you have it in your local directory which is what makes it work for you)

Reply:
nopes ... I don't have it in my local dir. It works because I in the config.py folder look at the join.sh file and update the path to the file - so that it points to where it's being generated, which is underneath the specific cluster folder in e.g. `~/.kcli/clusters/karmabs-cluster/join.sh` - so it should scale.